### PR TITLE
Support scala-2.12.0-RC2

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

File: project/build.scala [hunk 1]
```diff
@@ -37,20 +36,28 @@ object ApplicationBuild extends Build {
...
+    )},
 
     fork in run := true,
 
-    libraryDependencies += specs2 % "test"
+    libraryDependencies += specs2Core(scalaVersion.value) % "test"
   )
 
-  lazy val specs2 =    "org.specs2"    %% "specs2"    % "2.4"
+  def specs2Core(sv: String) = "org.specs2" %% "specs2-core" % "3.8.5.1"
```
Comment (bryce-anderson) [inline project/build.scala]: Is the `sv` param used somewhere I cant see right now?

**Review by bryce-anderson — APPROVED** [review]
Looks good, though I don't know what the `sv` param is for in the `specs2Core` function.

**Comment by bryce-anderson** [comment]
Should also change the title of the PR since its not 2.12-RC2 you're targeting.

File: project/build.scala (hunk 1 above)
Comment (rossabaker) [reply project/build.scala]: It became a function of Scala version back in M4, but no longer needs to be.

**Pull request merged**